getting_started/access_your_services.adoc

```diff
@@ -180,17 +223,94 @@ Endpoints:                <none>
 Session Affinity:         None
 External Traffic Policy:  Cluster
 ----
+endif::[]
+
+ifdef::openshift-arho[]
+[[expose-tcp-services]]
+== Expose TCP Services
+
+// TODO Need to update this with the Azure-specific load balancer info
+
+{product-title} xref:../architecture/networking/routes.adoc#architecture-core-concepts-routes[routes] expose applications by proxying traffic through HTTP/HTTPS(SNI)/TLS(SNI)
+to xref:../architecture/core_concepts/pods_and_services.adoc#architecture-core-concepts-pods-and-services[pods and services]. A link:https://kubernetes.io/docs/concepts/services-networking/#loadbalancer[LoadBalancer]
+service creates an AWS Elastic Load Balancer (ELB) for your {product-title} cluster, enabling direct TCP access to applications exposed by your LoadBalancer service.
+
+[NOTE]
+====
+LoadBalancer services require an additional purchase. Contact your sales team if you are interested in using LoadBalancer services for your {product-title} cluster.
+====
+
+=== Check your LoadBalancer Quota
+
+By purchasing LoadBalancer services, you are provided with a quota of LoadBalancers available for your {product-title} cluster.
+
+----
+$ oc describe clusterresourcequota service-loadbalancers
+Name:       service-loadbalancers
+Labels:     <none>
+Annotations:    <none>
+Resource        Used    Hard
+--------        ----    ----
+services.loadbalancers  0   4
+----
+
+=== Expose TCP Service
+
+You can expose your applications over an external LoadBalancer service, enabling access over the public Internet.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=lb-service
+service/lb-service exposed
+----
+
+=== Create an Internal-Only TCP Service
+
+You can alternatively expose your applications internally only, enabling access only through AWS VPC Peering or a VPN connection.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=internal-lb --dry-run -o yaml | awk '1;/metadata:/{ print "  annotations:\n    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0" }' | oc create -f -
+service/internal-lb exposed
+----
+
+=== Use your TCP Service
+
+Once your LoadBalancer service is created, you can access your service by using the URL provided to you by {product-title}.
+The `LoadBalancer Ingress` value is a URL unique to your service that remains static as long as the service is not deleted.
```

Comment:
You can specify an Azure DNS label for the service using the `service.beta.kubernetes.io/azure-dns-label-name` annotation